backend/src/appointment/templates/email/confirm_email.jinja2

```diff
@@ -1,17 +1,30 @@
 {% extends "includes/base.jinja2" %}
 {# Code begins! #}
 {% block introduction %}
-  <div style="color: {{ colour_text_base }}; text-align: center; line-height: 16px;">
+  <div style="color: {{ colour_text_base }}; text-align: center; line-height: 16px;margin-bottom: 32px;">
     <p style="font-weight: 700; margin: 0;">
       {{ l10n('confirm-email-mail-html-heading') }}
     </p>
   </div>
 {% endblock %}
 {% block call_to_action %}
-  <p>
-    {{ l10n('confirm-email-mail-html-body') }}
+  <center>
```

Comment:
Terrifying.

Reply:
Haha yes, writing markup for mail clients is sometimes like a history lesson on the origins of HTML 😂 Or writing a website for the Netscape Navigator.